### PR TITLE
Node 994 track when posts and comments have been edited

### LINKED ISSUE #994: Track when posts and comments have been edited
- [ ] Add an edited_at column to the posts table, and to the comments table, that are date times
- [ ] Add edited_at to the fields that can be added, updated and returned for those models

### PR DESCRIPTION
**Pull request by KevinTriplett**
Closes #994 

[Hylo-Evo PR 1648](https://github.com/Hylozoic/hylo-evo/pull/1648) depends on this PR

### CODE REVIEW

File: api/models/post/updatePost.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 import updateChildren from './updateChildren'
+import { isEqual } from 'lodash'
 import {
   updateGroups,
   updateAllMedia,
   updateFollowers
 } from './util'
 
-export default function updatePost (userId, id, params) {
+export default async function updatePost (userId, id, params) {
```
Comment (thomasgwatson) [inline api/models/post/updatePost.js:11]: Not sure why this async is needed? DIdn't see any awaits here or is it needed because of the tests?

**Review by thomasgwatson — APPROVED** [review]
lgtm

**Review by tibetsprague — APPROVED** [review]

File: api/models/post/updatePost.js (hunk 1 above)
Comment (KevinTriplett) [reply api/models/post/updatePost.js]: Good catch -- I removed the async and fixed the test

**Comment by KevinTriplett** [comment]
@tibetsprague @thomasgwatson I'm struggling to get circleci to update. It shows passed on the circleci site. I need to go to sleep, thanks for any help on this...

**Comment by KevinTriplett** [comment]
Nevermind -- I had joined the project at circleci and apparently "report on each commit" was defaulted off :b

**Pull request merged**